### ISSUE TITLE
Incorporate decreases into WB results

### ISSUE THREAD

**Issue by nicholst** [NISOx-BDI/SwE-toolbox#92]
Presently decreases of t-contrasts are saved as contrast 2 in WB mode but are not accessible via the user interface.

Modify the GUI so that negative results are accessible.  Seems like the easiest/best way to address this to add a line to the Results batch interface, something like:

> Use decreases of t-contrast (WB only)
> Yes
> No

@TomMaullin Does this seem like a reasonable approach?

**Comment by TomMaullin** [comment]
Hi @nicholst , Yes I was thinking something along the lines of the results batch window giving options such as the following:

```
- Contrast Name: <Manually entered>
- Contrast Type: [Activation| Deactivation] (only displayed if T contrast)
- Apply masking: [none|contrast|image]
...
```

What do you think of this layout?

**Comment by nicholst** [comment]
This looks good! Go for it!

**Comment by nicholst** [comment]
This is addressed by PR #95
